tests/tests.toml

```diff
@@ -374,6 +374,11 @@ filename = "../integration_tests/print_01.py"
 c = true
 cpp = true
 
+[[test]]
+filename = "../integration_tests/generics_01.py"
+asr = true
+llvm = true
```

Comment:
No need to test `llvm` here. Just integration tests and ASR is enough.

Reply:
I've removed them in the latest commit.